components/RegistrationForm.js

```diff
@@ -0,0 +1,18 @@
+const React = require('react');
+
+class RegistrationForm extends React.Component {
+  render() {
+    return (
+      <form>
+        <label>Username</label>
+        <input type="text" />
+        <label>Password</label>
+        <input type="password" />
+
```

Comment:
Is this space intended? Not sure about this, but it is it better to use a <br/>?